src/plugins/ini/README.md

```diff
@@ -1,18 +1,18 @@
-- infos = Information about the ini plugin is in keys below
-- infos/author = Thomas Waser <thomas.waser@libelektra.org>
-- infos/licence = BSD
+- infos=Information about the ini plugin is in keys below
```

Comment:
I do not think that it is a good idea to remove these spaces in the plugin contract, since the function [`generate_readme`](https://github.com/ElektraInitiative/libelektra/blob/master/cmake/Modules/LibAddMacros.cmake#L598-L754) seems to require them.